index.html

```diff
@@ -0,0 +1,335 @@
+                    </div>
+                    <h3 class="work-card-title ">Multi-Post<br>Stories</h3>
+                    <div class="work-card-dot ">
+                        <div class="card-text-main">CANOPY</div>
+                        <div class="work-card-dot-s"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">Back End Dev</div>
+                        <div class="work-card-dot-em2"></div>
+                        <div class="work-card-dot-list">
+                            <div class="card-icon ">
+                            </div>
+                        </div>
+                        <div class="work-card-dot-em"></div>
+                        <div class="card-text-main ">2015</div>
+                    </div>
+                    <h2 class="card-subtext ">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </h2>
+
+                    <ul class="grid">
+                        <li class="item">html</li>
+                        <li class="item">css</li>
+                        <li class="item">javascript</li>
+                    </ul>
+                    <div class="card-button ">
+                        <button type="button" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <section class="About ">
+            <div class="Aboutme-head ">
+                <div class="Aboutme-body ">
+                    <div class="main-title ">
+                        <h3 class="body-title">
+                            About<br>Myself</h3></div>
+            <p class="About-text ">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <h5 class="mini-text-abt ">LET’S CONNECT</h5>
+            <div class="abt-social-media ">
+                <ul class="media-icon ">
+                    <li><a href="#" class="fa fa-twitter"></a></li>
+ 
+                     <li><a href="#" class="fa fa-facebook-f"></a></li>
+ 
+                     <li><a href="#" class="fa fa-linkedin"></a></li>
+ 
+                     <li><a href="#" class="fa fa-github"></a></li>
+ 
+                     <li><a href="#" class="fa fa-angellist"></a></li>
+                    </ul>
+                  </div>
+            <div class="card-button ">
+              <button type="button" class="button1">Get my resume</button>
+            </div>
+            <div class="progrm-stack ">
```

Comment:
It seems you didn't quite understand the suggestion from the previous reviewer. According to the evaluation guidelines, this section should be rendered using `<ul>` and `<li>` tags (just like you did for the list of languages!). Kindly fix this.

<img width="679" alt="Screen Shot 2022-05-27 at 17 06 20" src="https://user-images.githubusercontent.com/82426682/170794970-d6eb3117-0869-46bd-a467-277b62ab6312.png">